### PR TITLE
Appimage

### PR DESCRIPTION
**Pull request by Ersikan**

### CODE REVIEW

File: libcommon/CMakeLists.txt [hunk 1]
```diff
@@ -13,29 +13,29 @@ if(NOT logger_POPULATED)
...
+#     set(MI_BUILD_OBJECT
+#         OFF
+#         CACHE INTERNAL "" FORCE
+#     )
+#     set(MI_BUILD_SHARED
+#         OFF
+#         CACHE INTERNAL "" FORCE
+#     )
+#     add_subdirectory(${mimalloc_SOURCE_DIR} ${mimalloc_BINARY_DIR})
+# endif()
```
Comment (zcorniere) [inline libcommon/CMakeLists.txt:38]: Why commenting mimalloc ?

File: .clang-format [hunk 2]
```diff
@@ -53,4 +53,5 @@ SpacesInContainerLiterals: false
 SpacesInParentheses: false
 SpacesInSquareBrackets: false
 UseTab: Never
-...
+IndentRequiresClause: false
+RequiresClausePosition: OwnLine
```
Comment (zcorniere) [inline .clang-format:57]: Sorry but llvm 15 is not available on manjaro for now, preventing others to work on the project.

Comment (Ersikan) [reply .clang-format]: I reformated using clang-format 14

File: libcommon/CMakeLists.txt (hunk 1 above)
Comment (Ersikan) [reply libcommon/CMakeLists.txt]: I reenabled by default

**Review by zcorniere — APPROVED** [review]

**Pull request merged**